### DISCUSSION TITLE
Intégration Prestashop google Analytics

### DISCUSSION THREAD

**Discussion by pierrickmartinet** [AmauriC/tarteaucitron.js#781] [Q&A]
Bonjour,

En ce qui concerne l'intégration sur Prestashop, comment est ce possible de remplacer de script de google analytics par le script de tarteaucitron lorsqu'on passe par un module ?

merci d'avance :)

**Comment by pierretux** [comment]
Bonjour,

Je pense que cela se passe comme pour les autres CMS : on supprime le module et report le code GA- dans tarteaucitron.

Cdt
